src/udisksdaemonutil.c

```diff
@@ -56,6 +56,22 @@
  * Various utility routines.
  */
 
+
+/**
+ * udisks_string_wipe_and_free:
+ * @string: A string with potentially unsafe content or %NULL.
+ *
+ * Wipes the buffer and frees the string.
+ */
+void udisks_string_wipe_and_free (GString *string)
+{
+  if (string != NULL)
+    {
+      memset (string->str, '\0', string->len);
+      g_string_free (string, TRUE);
+    }
+}
+
```

Comment:
Oh, sure, it was the same case with me. I don't know why GH does this sometimes. Thanks for the explanation and feel free to ignore the comment/suggestion. :)